### ISSUE TITLE
APB join not working with TTN / OTA works fine

### ISSUE THREAD

**Issue by ssozonoff** [GrumpyOldPizza/ArduinoCore-stm32l0#34]
Hi,

Tried your APB example to TTN but can get it working. TTN is working great.
Any ideas ?

Kind thanks,
Serge

**Comment by GrumpyOldPizza** [comment]
Does the OTA example work for you then, after of course entering your keys ? Did you change the region properly to match your geographic location ?

**Comment by ssozonoff** [comment]
OTA works. Normally everything should be fine but let me try again this evening.
